packages/calcite-design-tokens/src/tokens/semantic/space.json

```diff
@@ -5,161 +5,267 @@
         "xxs": {
           "value": "{core.size.default.4}",
           "type": "spacing",
+          "description": "deprecated",
           "attributes": {
             "category": "spacing"
-          },
-          "description": "deprecated"
+          }
         },
         "xs": {
           "value": "{core.size.default.6}",
           "type": "spacing",
+          "description": "deprecated",
```

Comment:
I was going to suggest adding the recommended prop here, but I think we need to revisit whether we need to include these messages in the output. The TypeScript declaration files might be an exception, but I don't think IntelliSense picks up these comments and I'm not sure developers are looking in the build output for this info and whether that's practical.